soc/nxp/mcx/mcxn/Kconfig

```diff
@@ -35,9 +48,9 @@ config SOC_MCXN236
 
 if SOC_SERIES_MCXN
 
-if SOC_MCXN947
+if SOC_MCXN947 || SOC_MCXN547
 config SECOND_CORE_MCUX
```

Comment:
newline

Reply:
What do you mean?
Formatting issue? or duplicate whole code in two different conditions `SOC_MCXN947` and `SOC_MCXN547`?

Reply:
```suggestion
if SOC_MCXN947 || SOC_MCXN547

config SECOND_CORE_MCUX
```

Reply:
Done.